tracer/src/Datadog.Trace/AppSec/Rcm/AsmRemoteConfigurationProducts.cs

```diff
@@ -5,17 +5,25 @@
 
 #nullable enable
 
-using Datadog.Trace.Configuration;
+using System.Collections.Generic;
 
-namespace Datadog.Trace.AppSec;
+namespace Datadog.Trace.AppSec.Rcm;
 
 internal static class AsmRemoteConfigurationProducts
 {
     public static AsmFeaturesProduct AsmFeaturesProduct { get; } = new();
 
     public static AsmDataProduct AsmDataProduct { get; } = new();
 
-    public static AsmDDProduct AsmDDProduct { get; } = new();
+    public static AsmDdProduct AsmDdProduct { get; } = new();
 
     public static AsmProduct AsmProduct { get; } = new();
+
+    public static Dictionary<string, AsmRemoteConfigurationProduct> GetAll() => new()
```

Comment:
This shouldn't be a method but a static field. Because each time you would call the method, you would create a new Dictionary (you don't but you see what I mean).
Also you should use a ReadOnlyDictionary if I'm not mistaken